src/redux/auth/thunk.ts

```diff
@@ -66,16 +66,22 @@ export const updateUserExperience = createAsyncThunk<
   }
 });
 
-export const getUserBadgesThunk = createAsyncThunk<
-  Badge[],
+export const resetThunk = createAsyncThunk<
   void,
+  ResetPayload,
   {rejectValue: SerializedError}
->('user/getBadges', async (_, {rejectWithValue}) => {
+>('auth/reset', async ({email}, {rejectWithValue}) => {
   try {
-    const {data} = await api.user.getBadges();
+    const {data} = await api.auth.resetPassword({email});
```

Comment:
You will not get any data from this api. You can write this line without declaration of constant variable. Just “await api…”. And after that just redirect to necessary screen. If request fails, navigation will not call because it will move your code to “catch” block